crates/cxx-qt-lib/src/gui/qpainter.rs

```diff
@@ -119,10 +124,19 @@ mod ffi {
         #[rust_name = "draw_polyline"]
         fn drawPolyline(self: Pin<&mut QPainter>, points: &QPolygon);
 
+        /// Draws the current rectangle with the current pen and brush.
+        #[rust_name = "draw_rect"]
```

Comment:
Maybe rename this to `draw_rect_f`?
We probably don't need `draw_rect`, but that leaves us some naming wiggle room to add it.